src/pages/HomePage/hooks.ts

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from "react";
+import { get, size } from "lodash";
+import {
+  useQueryWithClient,
+  useDeskproLatestAppContext,
+} from "@deskpro/app-sdk";
+import { getEntityListService } from "../../services/deskpro";
+import { getDevicesService } from "../../services/lansweeper";
+import { enhanceDevices } from "../../utils";
+import { QueryKey } from "../../query";
+import type { DeviceType, UserContext } from "../../types";
+
+type UseLinkedDevices = () => {
+  isLoading: boolean;
+  devices: DeviceType[];
+};
+
+const useLinkedDevices: UseLinkedDevices = () => {
+  const { context } = useDeskproLatestAppContext() as { context: UserContext };
```

Comment:
We shouldn't be typing here. The SDK should be providing proper types and if not, since we own that SDK, we should fix it.

Reply:
I created a story to fix it in SDK https://app.shortcut.com/deskpro/story/158090/app-sdk-should-fix-context-typings

Reply:
Cool beans, but can we add more to the story. A single screenie doesn't say much as is missing the larger picture. Whilst we're here, might as well say why you typed it what you expected the SDK to do instead.

Reply:
Added more details